openquake/commonlib/calculators/event_based.py

```diff
@@ -338,11 +342,14 @@ def post_execute(self, result):
         """
         nc = self.rlzs_assoc.csm_info.num_collections
         sescollection = [{} for col_id in range(nc)]
+        tags = []
         for trt_id in result:
             for sr in result[trt_id]:
                 sescollection[sr.col_id][sr.tag] = sr
+                tags.append(sr.tag)
         logging.info('Saving the SES collection')
         with self.monitor('saving ses', autoflush=True):
+            self.tags = numpy.array(sorted(tags), (str, 100))
```

Comment:
Please add a comment about the possibility that the string will be truncated if longer than 100